cellfinder/core/tools/threading.py

```diff
@@ -248,8 +248,8 @@ def get_msg_from_thread(self, timeout: Optional[float] = None) -> Any:
             ...         # do something with the msg
             ...         pass
             ... except ExecutionFailure as e:
-            ...     print(f"got exception {type(e.__cause__)}")
-            ...     print(f"with message {e.__cause__.args[0]}")
+            ...     logger.error(f"got exception {type(e.__cause__)}")
```

Comment:
I don't think the `logger` variable is imported in this file, so we will need to do that first.

Also, you've mispelt `logger` as `looger` below